I made a pleasant and understandable README.md

rodukov opened the pull request:
🎁 **The fact is that the main repository still does not have a detailed installation guide. I made it.**

rodukov committed:
README.md

rodukov committed:
README.md

Bitwise-01 reviewed (changes requested):
Make it look prettier, use a markdown linter or something

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov committed:
Update README.md

rodukov commented:
> Make it look prettier, use a markdown linter or something

🎉 **I updated the README.md again**
🔻 I simplified the README.md;
🔻 Emojis added;
🔻 Added badges;
🔻 I moved the finished dictionary(`pass.lst`) to the `databases` folder.

Bitwise-01 commented:
Great job!
